README.md

```diff
@@ -9,218 +8,20 @@ Nextcloud server packaged as a snap. It consists of:
-      Note that it requires `sudo`.
-- `nextcloud.mysqldump`:
-    - Dump Nextcloud database to stdout. You should probaby redirect its output
-      to a file. Note that it requires `sudo`.
-- `nextcloud.enable-https`:
-    - Enable HTTPS via self-signed certificates, Let's Encrypt, or custom
-      certificates. HTTP will redirect to HTTPS. Non-custom certificates will
-      automatically be kept up-to-date. See `nextcloud.enable-https -h` for more
-      information. Note that it requires `sudo`.
-- `nextcloud.disable-https`:
-    - Disable HTTPS (does not remove certificates). Note that it requires
-      `sudo`.
-- `nextcloud.manual-install`:
-    - Manually install Nextcloud instead of visiting it in your browser. This
-      allows you to create the admin user via the CLI. Note that it requires
-      `sudo`.
-- `nextcloud.export`:
-    - Export data suitable for migrating servers. By default this includes the
-      Nextcloud database, configuration, and data. See `nextcloud.export -h` for
-      more information. Note that it requires `sudo`.
-- `nextcloud.import`:
-    - Import data exported from another Nextcloud snap instance (via
-      `nextcloud.export`). By default this imports the database, config, and
-      data. See `nextcloud.import -h` for more information. Note that it
-      requires `sudo`.
-
-
-## Where is my stuff?
-
-- `$SNAP_DATA` (`/var/snap/nextcloud/current/` by default)
-    - Logs (Apache, PHP, MySQL, Redis, and Nextcloud logs)
-    - Keys and certificates
-    - MySQL database
-    - Redis database
-    - Nextcloud config
-    - Any Nextcloud apps installed by the user
-- `$SNAP_COMMON` (`/var/snap/nextcloud/common/` by default)
-    - Nextcloud data
-
-
-## Hacking
-
-If you change something in the snap, build it, install it, and you can run a
-suite of acceptance tests against it. The tests are written in ruby, using
-capybara and rspec. To run the tests, you first need to install a few
-dependencies:
-
-    $ sudo apt install gcc g++ make qt5-default libqt5webkit5-dev ruby-dev zlib1g-dev
-    $ sudo gem install bundle
-    $ cd tests/
-    $ bundle install
-
-Additionally, if you do not have X configured, install the following for a
-'fake' X server.
-
-    $ sudo apt install xvfb
-
-Make sure the snap has a user called "admin" with password "admin" (used for
-login tests):
-
-    $ sudo nextcloud.manual-install admin admin
-
-And finally, run the tests:
-
-    $ cd tests/
-    $ rake test
+Type the following in your terminal:
+```
+sudo snap install nextcloud
+```
+**That's it, you're done!**
 
-[1]: https://github.com/nextcloud/nextcloud-snap/wiki/Release-strategy
+* [**Configure**](https://github.com/nextcloud-snap/nextcloud-snap/wiki/configure-Nextcloud-snap) Nextcloud and regain control over your data.
+* [**Wiki**](https://github.com/nextcloud-snap/nextcloud-snap/wiki/Nextcloud-snap-wiki) for more details.
+* [**Getting help**](https://github.com/nextcloud-snap/nextcloud-snap/wiki/Getting-help)
```

Comment:
Can we add another link to "How to develop" or something along those lines? I'd like to not lose those hacking steps ^ .